docs/logging-format.md

```diff
@@ -228,6 +228,35 @@ and TLS probe points.
 | `tls::key_exchange_algorithm` | uint16     | Key exchange mode: ECDHE(0), DHE(1), PSK(2), ECDHE-PSK(3), DHE-PSK(4)                            |
 | `tls::group`                  | uint16     | Groups used in the handshake (as in IANA [registry][iana-tls-supported-groups])                  |
 
+##### SSH context names
+
+| name                   | description                 |
+|------------------------|-----------------------------|
+| `ssh::handshake_client`| SSH handshake for client    |
+| `ssh::handshake_server`| SSH handshake for server    |
+| `ssh::client_key_sign` | SSH client key proof        |
+| `ssh::server_key_sign` | SSH server key proof        |
+| `ssh::key_exchange`    | SSH key exchange            |
+
+##### SSH keys
+
+All the keys except `rsa_bits` have `string` type.
+
+| key                             | description                                      | example                                                     |
+|---------------------------------|--------------------------------------------------|-------------------------------------------------------------|
+| `ssh::ident_string`             | Software identity string                         | `SSH-2.0-OpenSSH_8.8`                                       |
```

Comment:
RFC 4253 describes this as "identification string" so I would stick with the RFC naming:
```suggestion
| `ssh::ident_string`             | Software identification string                         | `SSH-2.0-OpenSSH_8.8`                                       |
```
as mentioned in the personal message, this is not clear if this is client or server identification string (initial version had both). I think we want to report both.

Reply:
this is handshake so yes, we have the context if this is client or server, but we use both of them in the kex.

Reply:
I added a clarification message, could you please confirm that it's clear now?

Reply:
Looks good.